src/QualificationRunner.Core/Domain/SimulationParameterRef.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OSPSuite.Core.Qualification;
+
+namespace QualificationRunner.Core.Domain
+{
+   public class SimulationParameterRef : IReferencingSimulation

```

Comment:
same here
This was moved by mistake. It did not belong in core as this is only read by the QualificationRunner